src/assets/usage-examples-code/graphics/clear_screen/clear_screen-1-background-color.cs

```diff
@@ -0,0 +1,9 @@
+using static SplashKitSDK.SplashKit;
+
+OpenWindow("House Drawing", 800, 600);
```

Comment:
Code executes without issue. The only thing is the C# code needs to be changed as you've used the title of the window "House Drawing" rather than "Back Ground" like you did in the C++ code. I would also recommend changing it to Background just for grammar's sake.

Reply:
Thanks! I have changed the House Drawing in c# to Back Ground.